### ISSUE TITLE
Support Zipped TIFF Files

### ISSUE THREAD

**Issue by DanielJDufour** [GeoTIFF/georaster#18]
**Is your feature request related to a problem? Please describe.**

Users sometimes have GeoTIFFs in zipped tiff file.  For example, a file name could be something like `ndvi.tif.zip` 

**Describe the solution you'd like**

Users should be able to be able to pass a (array)buffer of the zipped file to georaster, just as they can pass a buffer of an unzipped GeoTIFF.  There may be a way to identify whether an buffer represents a zip file by looking for a magic number (i.e. byte identifier).  There appears to be a few packages on NPM that do this.  Therefore, the user won't have to specify the file type.  They can just directly pass in a buffer and that's it.

**Describe alternatives you've considered**

None at the moment

**Additional context**

Basically, GeoTIFFs can get large and so people zip compress them.  When people use mobile devices, it can be especially annoying and difficult to unzip tiff files.  We want people to be able to use geotiff.io on mobile devices with .tiff.zip files.

**Comment by Rdataflow** [comment]
State of the art is to publish compressed TIFF files i.e. using LZW, DEFLATE, ZSTD, LERC or LERC_ZSTD ...
see also https://kokoalberti.com/articles/geotiff-compression-optimization-guide/
and `COMPRESS` on https://gdal.org/en/latest/drivers/raster/cog.html#general-creation-options
thus let's rather update `geotiff.js` package to depend on latest version

with this compressed zip files are obsolete

cc @DanielJDufour
